dmoj/cptbox/_cptbox.pyx

```diff
@@ -183,16 +190,13 @@ def bsd_get_proc_fdno(pid_t pid, int fd):
 cdef class Debugger:
     cdef pt_debugger *thisptr
     cdef object on_return_callback
-    cdef int _getpid_syscall
-    cdef int _debugger_type
 
-    property type:
-        def __get__(self):
-            return self._debugger_type
+    def __cinit__(self):
+        self.thisptr = new pt_debugger()
 
-    property getpid_syscall:
+    property skip_syscall_id:
```

Comment:
Maybe `noop_syscall_id` is more truthful, since the syscall itself isn't skipped.